docs/postrelease.md

```diff
@@ -8,6 +8,12 @@ This checklist is to be run after cutting a release.
   - Example: https://github.com/gravitational/teleport/pull/7033
 - [ ] Create PR to update default AMI versions in Makefile and AMIs.md under https://github.com/gravitational/teleport/blob/master/assets/aws
   - Example command: `TELEPORT_VERSION=6.2.0 make -C assets/aws create-update-pr`
+- [ ] Release Teleport Plugins
```

Comment:
I would just link to the `teleport-plugins` repo here, so it's clear that this make command must run there, not in `teleport`.